packages/jazz-svelte/src/lib/tests/Provider.svelte.test.ts

```diff
@@ -1,120 +0,0 @@
-import { describe, it, expect, vi, beforeEach } from 'vitest';
```

Comment:
Going to cover this kind of components with a browser-based test, otherwise it's very hard to maintain.